FirebaseMessaging/Sources/Public/FirebaseMessaging/FIRMessagingExtensionHelper.h

```diff
@@ -14,8 +14,11 @@
  * limitations under the License.
  */
 
-#if __IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_10_0 || \
-    __MAC_OS_X_VERSION_MAX_ALLOWED >= __MAC_10_14
+#import <Foundation/Foundation.h>
+
+@class UNMutableNotificationContent, UNNotificationContent;
```

Comment:
will anything below iphone 10 and mac 10.14 report errors recognizing this classes?

Reply:
These are forward declarations, so they'll work on any OS+version combination. I'm not sure what's causing the tests to fail though.